components/lcms2/Makefile

```diff
@@ -60,3 +65,7 @@ BUILD_PKG_DEPENDENCIES = $(BUILD_TOOLS)
 
 include ../../make-rules/depend.mk
```

Comment:
Just for my understanding: any component I find that is using either

`include ../../make-rules/prep.mk`

or

`include $(WS_TOP)/make-rules/prep.mk`

should be changed to instead use

`include $(WS_MAKE_RULES)/prep.mk`

(for configure.mk, ips.mk, etc. too).